src/actions/postOnFacebook.ts

```diff
@@ -0,0 +1,17 @@
+import { withActionOutput } from '../utils';
+
+import WFWorkflowAction from '../interfaces/WF/WFWorkflowAction';
+
+/**
+ * Shares the input on Facebook.
```

Comment:
Could you edit this line to the following, please?
```
 * Post on Facebook Action. Shares the input on Facebook.
```

This should be added also to all the other actions you submitted.
In the end, each action should have a comment in this format:
```
 * {{Action name as shown in app}} Action. {{Description as shown in app}}.
```
This format will help getting the shown name in the next documentation website. Thanks!

Reply:
Added